falcon/response_helpers.py

```diff
@@ -60,3 +62,18 @@ def format_range(value):
             str(value[0]) + '-' +
             str(value[1]) + '/' +
             str(value[2]))
+
+
+class TimezoneGMT(tzinfo):
```

Comment:
I think this class might be better placed in `util/misc.py`. In the future, it might be useful outside of Response.